tpr/tpr.go

```diff
@@ -297,3 +313,20 @@ func (t *TPR) waitInit(retry backoff.BackOff) error {
 	}
 	return microerror.Maskf(err, "requesting TPR %s", t.name)
 }
+
+func (t *TPR) StartMetrics() {
+	go func() {
+		ticker := time.NewTicker(time.Second * 5)
+
+		for range ticker.C {
+			list, err := t.listFunc(apismetav1.ListOptions{})
+			if err != nil {
+				t.logger.Log("error", "could not list objects for metrics", "message", err.Error())
+				continue
+			}
+
+			// TODO: set metrics.
```

Comment:
Issue here is that `listFunc` returns a `runtime.Object`, so I'm not certain how to get a count of the total number of TPOs.